packages/core/src/command/patient-import/api/shared.ts

```
@@ -0,0 +1,53 @@
+    const res = await executeWithNetworkRetries(functionToRun, {
+      ...defaultOptionsRequestNotAccepted,
+    });
+    return res;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  } catch (error: any) {
+    log &&
+      log(
+        `${messageWhenItFails}. Additional info: ${JSON.stringify(
+          additionalInfo
+        )}. Cause: ${errorToString(error)}`
+      );
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)